certification/runtime/runtime.go

```diff
@@ -15,6 +15,7 @@ type Config struct {
 	Mounted        bool
 	Bundle         bool
 	Scratch        bool
+	Privileged     bool
```

Comment:
I might be missing it; what does this do for us?

I don't mind it being here... because at some point I think we should migrate back to a single-point-of-configuration type workflow, which doesn't exist. But we haven't had conversations about that, and I'm not sure I'm seeing where this is used.

Reply:
Right now we aren't using this, I feel its better to add it now incase we need to use it. But if others feel it can be removed, that's fine as well.

Reply:
I would say leave it out. We did this for scratch, because we needed to skip the rpmmanifest. We don't need to know about it in the engine.

Reply:
I removed this.